pkgs/development/libraries/libaosd/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub, pkgconfig, cairo, pango,
+  libX11, libXcomposite, autoconf, automake }:
+
+stdenv.mkDerivation rec {
+  version = "0.2.7-9-g177589f";
+  name = "libaosd-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "atheme-legacy";
+    repo   = "libaosd";
+    rev    = "${version}";
+    sha256 = "1cn7k0n74p6jp25kxwcyblhmbdvgw3mikvj0m2jh4c6xccfrgb9a";
+  };
+
+  buildInputs = [autoconf automake pkgconfig cairo pango libX11 libXcomposite];
+
+  prePatch = ''
```

```suggestion
  preConfigure = ''
```
would be more sensible